.circleci/config.yml

```diff
@@ -337,9 +337,10 @@ jobs:
       - run:
           name: Unit test coverage
           command: make unit-tests-with-cover
-      - run:
-          name: Upload unit test coverage
-          command: bash <(curl -s https://codecov.io/bash) -F unit
+# DISABLE CODECOV UNTIL THE SCRIPT IS AUDITED AND WE ARE CERTAIN IT IS OK TO TO EXECUTE IT.
```

Comment:
For reference, the github action now verifies checksums

https://github.com/codecov/codecov-action/pull/282

Still not the best it can be since it uses a checksum from the github repo but at least that is separate from the `codecov.io` site itself. But migrating to github actions will definitely allow some more safety for it than just the curl.